mail/dovecot2/Portfile

```diff
@@ -43,9 +57,18 @@ patch.pre_args      -p1
 patchfiles          patch-doc-example-config-conf.d-10-master.conf.diff \
                     patch-src-master-master-settings.c.diff
 
-checksums           rmd160  37230c065284899fa107ab1c91831ca154f230ac \
-                    sha256  3b61c128f3ef947f5d3e8b1113c321d0597600a3102a8e5b405484a7c049eb51 \
-                    size    3653354
+checksums           patch-doc-example-config-conf.d-10-master.conf.diff \
```

Comment:
The patchfiles are all usually put in the "files" directory beside the Portfile, and they don't have checksums. There is only one port I know of, an old gcc-4.2 version, that downloads a (huge) patchfile.

Reply:
It should not have reported the checksums for the patch files -- something is wrong there. Are the patchfiles actually in the files directory?

Reply:
yes